package.json

```diff
@@ -44,11 +44,14 @@
     "serve:dbg": "npm run dev:dbg",
     "preview": "vite preview",
     "test:build": "cd src/core && cargo test --release --features webgl2",
-    "test:unit": "vitest run",
+    "test:playwright": "npx playwright test",
+    "test:update-snapshots": "npx playwright test --update-snapshots",
     "doc": "jsdoc -c jsdoc.json src/js src/js/shapes && cp aladin-logo.png docs/",
     "doc:dev": "npm run doc && open docs/index.html"
   },
   "devDependencies": {
+    "@playwright/test": "^1.45.2",
+    "@types/node": "^20.14.11",
     "happy-dom": "^10.11.0",
     "jsdoc": "^4.0.2",
     "vite": "^4.3.8",
```

Comment:
I think you can remove vite-related dependencies

Reply:
I think that vite is used for the dev server, but I will test to remove it.

Reply:
You're right. But you might be able to remove the dependencies added in this commit https://github.com/cds-astro/aladin-lite/commit/402e27001581d287e7c6471a76c249148fce8dad